sisyphus/cleaner.py

```diff
@@ -280,8 +295,17 @@ def cleanup_jobs():
             job._sis_cleanup()
 
 
-def cleanup_keep_value(min_keep_value, load_from: str = "", mode: str = "remove"):
-    """Go through all jobs in the current graph to remove all jobs with a lower keep value that the given minimum"""
+def cleanup_keep_value(
+    min_keep_value, load_from: str = "", mode: str = "remove", filter_affected: Optional[List[str]] = None
+):
+    """Go through all jobs in the current graph to remove all jobs with a lower keep value that the given minimum
+
+    :param min_keep_value: Remove jobs with lower keep value than this
+    :param load_from: File name to load list with used directories
+    :param mode: Cleanup mode ('remove', 'move', or 'dryrun')
+    :param filter_affected: Defines what substrings should be printed when listing affected directories
```

Comment:
wow, I am very surprised by `filter_affected`. Because the thing that is affecting the files/directories in these functions is <being deleted>. So I expected the parameter `filter_affected` to filter the folders that are deleted.

But as the docstring clearly states: the affecting only pertains to the being printed part, the folders would still be deleted.

Is this intended? Ok, by the PR title "Add Logging filtering" I assume yes.
But..
* I don't see why one would want to clean everything but only have a specific thing printed to log.
* I do see how one would want to clean up only specific kinds of jobs (e.g. clean up recognitions but keep all the trainings even if unused)

Suggestions:
* change the implementation so that `filter_affected` filters what is cleaned OR
* change the name of the parameter to something like filter_printed / logging_filter / ..

Reply:
So for me just printing was desired, since I usually have like 10k jobs that want to be cleaned and I just want to print the „longer“ jobs. 
But I can change the name.